packages/cli/openapi-ir-to-fern/src/buildQueryParameter.ts

```diff
@@ -298,6 +311,8 @@ function getQueryParameterTypeReference({
                         generatedName: schema.generatedName,
                         value: oneOfSchema,
                         description: undefined,
+                        // TODO (rohin): not sure about this:
+                        availability: undefined,
```

Comment:
why not `schema.availability` ?

Reply:
Schema is arbitrary -- some of the schemas don't have availability. Coercion might lead to a runtime error, but might be able to do a property check here